### ISSUE TITLE
Texture over default mesh?

### ISSUE THREAD

**Issue by UmiAkai** [derekhe/xplane-map-enhancement-release#43]
This is a general question I have, why don't map enhancement simply apply texture over the X-plane's default terrain mesh? Instead using an outdated, not very gateway airport compatied and very hard to download very large base mesh?
For example, in default ZSPD, part of north parking area terrain is very deformed with basepack V2, while default mesh is nice-not very sloped but also not totally flat.
Also issues like I found in RJTT or VTSB etc. etc.

**Comment by derekhe** [comment]
Indeed, we must proceed in this manner to intercept some file reading. This is contingent on Ortho4XP, which requires an update for XPlane 12. Once improvements are made, I intend to update the base package, which may resolve this issue.
